hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerInfo.java

```diff
@@ -154,10 +154,12 @@ public ReplicationConfig getReplicationConfig() {
     return replicationConfig;
   }
 
```

Comment:
Quick question: Here JasonIgnore will make it like optional ? So that it will be still available in Ratis types

Reply:
In the json output from the command "replicationFactor" and "replicationType" will no longer be output, as they are duplicated inside replicationConfig: { type: ... factor ... }.

For a command like this, I don't think it makes sense to keep the old version now we have replicationConfig instead.

It would significantly complicate things to optionally output these fields in the json depending if we have Ratis or EC, as right now the JSON helper outputs all getters on the object, unless we ignore them as in this change.